build.sbt

```diff
@@ -384,9 +390,25 @@ lazy val engineStandalone = (project in engine("standalone/engine")).
   ).
   dependsOn(interpreter % "provided", standaloneUtil, httpUtils % "provided", testUtil % "it,test")
 
+lazy val standaloneDockerSettings = {
+  val workingDir = "/opt/nussknacker"
+
+  commonDockerSettings ++ Seq(
+    dockerEntrypoint := Seq(s"$workingDir/bin/nussknacker-standalone-entrypoint.sh", dockerPort.toString),
+    dockerExposedVolumes := Seq(s"$workingDir/storage", s"$workingDir/data"),
```

Comment:
The only storage is `/standaloneProcesses`, BTW maybe we should reconfigure it to some `/opt/standalone/processes` to be consistent with nk convention?

Reply:
Or maybe `/opt/processes` is enough? It's a path inside standalone container

Reply:
`"/opt/standalone/processes"` or `"/opt/nussknacker/processes"`? Or maybe ``"/opt/nussknacker-standalone/processes"`` ;)

Reply:
Don't we need `"$workingDir/storage"` for `logs` directory?

Reply:
`/opt/nussknacker/processes` is fine :+1: . Regards to storage - you've right, we have logs inside so it is ok. BTW logs are not stored there because we have only console appender (in nk and standalone) but lets it stay as it is. Maybe it is for some advanced configurations of logging.